packages/fscomponents/src/components/Accordion.tsx

```diff
@@ -82,6 +82,10 @@ export interface AccordionProps {
    * Function to be invoked when the accordion is opened. Animation will be disabled!
    */
   renderContent?: () => React.ReactNode;
+  /**
+   * Function to be invoked to check the state of the accordion. Takes a callback."
+   */
+  onStateChange?: (stateChanged: boolean) => void;
```

Comment:
callback function does not need to take stateChanged flag, because function will be called only when state is changed. Right?